modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/java.util.txt

```diff
@@ -574,8 +574,8 @@ class java.util.Collections {
   Comparator reverseOrder()
   Comparator reverseOrder(Comparator)
   void rotate(List,int)
-  void shuffle(List)
-  void shuffle(List,Random)
+  void shuffle(List) @nondeterministic
```

Comment:
This could run into the shuffle is called on a def problem for runtime resolution where we could possibly miss the randomized result. Though, given how loose we're being with what's cacheable, maybe we should ignore this specific instance for now.

Reply:
Oops! Never mind as these are static methods. Misread that as List would inherit these.